README.brazilian-portuguese.md

```diff
@@ -282,15 +282,14 @@ function someFunction()
 const count = 2;
 (function doSomething() {
   // do something amazing
-}());
+})();
```

Comment:
nit: Looks like this slipped through. Please undo this too.

Reply:
That one is actually a good change since the previous code wasn't correct 👍

Reply:
It's only a matter of preference. Both are correct.
```js
(function() {
  //body
}())
```
and 
```js
(function() {
  //body
})()
```
do the same thing.

The only reason I'm hesitating is 'coz if we make this change here, then all the other language READMEs need to be synced too. Also, [L291](https://github.com/i0natan/nodebestpractices/pull/358/files#diff-0dff8ae931792f86c957630a81d0e679R291) will need to be updated to match the same IIFE pattern.

I personally don't find it worth the change, but am not opposed to it if someone wants to update all the change-sites.

Reply:
Ah I see, let's change this back then 👍

Reply:
Resolving since the suggestion below takes care of it.